websocketOutportDriver/utils_test.go

```diff
@@ -0,0 +1,46 @@
+package websocketOutportDriver_test
+
+import (
```

Comment:
I do not like util_test.go
Maybe change it to subcomponents_test.go

Reply:
removed